client/components/common/tce-core/ElementDiscussion.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <v-menu
+    v-model="isVisible"
+    :close-on-content-click="false"
+    transition="slide-y-transition"
+    min-width="300"
+    attach offset-y left>
+    <template v-slot:activator="{ on: menu }">
+      <v-tooltip open-delay="800" left>
+        <template v-slot:activator="{ on: tooltip }">
+          <v-btn v-on="{ ...menu, ...tooltip }" icon>
+            <v-avatar v-if="unseenComments.length" size="18" color="secondary">
+              {{ unseenComments.length }}
+            </v-avatar>
+            <v-icon v-else color="primary" class="pr-1">mdi-forum-outline</v-icon>
+          </v-btn>
+        </template>
+        <span>Discussion</span>
```

Comment:
Let's improve the label by setting it to:

- `Post a comment` if there are no comments
- `View comments` if there are comments.